### DISCUSSION TITLE
[LSPSaga] How to show line number and relative number in finder float window?

### DISCUSSION THREAD

**Discussion by VincentChuck** [nvimdev/lspsaga.nvim#1247] [Q&A]
Is there an option to show the line number and relative line number in the floating window? 
E.g. `require('lspsaga').setup({finder = {number = true}}) ` in the setup function

**Comment by glepnir** [comment]
nope is there any reason that you want show  line number on finder?

**Reply by VincentChuck** [reply]
it makes it easier to keep track of which line you're at and navigate around when there're multiple results in the same file

**Reply by glepnir** [reply]
oh you mean the code line number right ?

**Reply by VincentChuck** [reply]
yepp, the circled area in my screenshot attached
![Screenshot 2023-08-17 at 2 37 53 PM](https://github.com/nvimdev/lspsaga.nvim/assets/40799382/70a1f80e-507d-4c70-b848-4fca41f6d278)

**Reply by glepnir** [reply]
make sense in some situation can add an option for that maybe.
